rdt/__init__.py

```diff
@@ -157,6 +164,9 @@ def _find_addons():
             warnings.warn(msg)
             continue
 
+        if isinstance(addon, ModuleType) and addon.__name__ not in sys.modules:
+            sys.modules[addon.__name__] = addon
```

Comment:
This will not work as expected because the `addon.__name__` is just the base name of the addon being added, it has to include the `addon_target` as well.
```python
target_sys_name = f'{addon_target}.{addon.__name__}'
sys.modules[target_sys_name] = addon
```

Reply:
The `addon.__name__` is actually the whole entry point name, but you're right it needs to be the target name added to the addon_name